mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -72,6 +93,36 @@ const addFavoriteWayUuid = async (
   await UserPreviewDAL.updateUserPreview(updatedUserPreview);
 };
 
+/**
+ * Add user uuid to favoriteForUserUuids array
+ */
+const addFavoriteForUserUuids = async (
+  way: WayPreview,
+  userUuid: string,
+  setWay: React.Dispatch<React.SetStateAction<WayPreview | undefined>>,
+) => {
+  const updatedFavoriteForUserUuids = [...way.favoriteForUserUuids, userUuid];
+  const updatedWay = createWayWithUpdatedFavorites(way, updatedFavoriteForUserUuids);
+
+  await WayPreviewDAL.updateWayPreview(updatedWay);
+  setWay(updatedWay);
+};
+
+/**
+ * Delete user uuid from favoriteForUserUuids array
```

Comment:
Maybe it is worth to tell for which model it is related

Reply:
Done.